python-avd/pyavd/schema/constants.py

```diff
@@ -3,26 +3,39 @@
 # that can be found in the LICENSE file.
 from pathlib import Path
 
-from yaml import safe_load
-
 REPO_ROOT = Path(__file__).parents[3]
-EOS_DESIGNS_SCHEMA_PATH = REPO_ROOT.joinpath("ansible_collections/arista/avd/roles/eos_designs/schemas/eos_designs.schema.yml")
-EOS_CLI_CONFIG_GEN_SCHEMA_PATH = REPO_ROOT.joinpath("ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml")
-
-with open(EOS_CLI_CONFIG_GEN_SCHEMA_PATH, encoding="UTF-8") as file:
-    EOS_CLI_CONFIG_GEN_SCHEMA = safe_load(file)
+EOS_DESIGNS_SCHEMA_DIR = REPO_ROOT.joinpath("ansible_collections/arista/avd/roles/eos_designs/schemas")
+EOS_DESIGNS_SCHEMA_PATH = EOS_DESIGNS_SCHEMA_DIR.joinpath("eos_designs.schema.yml")
+EOS_DESIGNS_FRAGMENTS_PATH = EOS_DESIGNS_SCHEMA_DIR.joinpath("schema_fragments")
+EOS_DESIGNS_PICKLED_SCHEMA_PATH = EOS_DESIGNS_SCHEMA_DIR.joinpath("eos_designs.schema.pickle")
+EOS_DESIGNS_JSONSCHEMA_PATH = EOS_DESIGNS_SCHEMA_DIR.joinpath("eos_designs.jsonschema.json")
 
-with open(EOS_DESIGNS_SCHEMA_PATH, encoding="UTF-8") as file:
-    EOS_DESIGNS_SCHEMA = safe_load(file)
+EOS_CLI_CONFIG_GEN_SCHEMA_DIR = REPO_ROOT.joinpath("ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas")
+EOS_CLI_CONFIG_GEN_SCHEMA_PATH = EOS_CLI_CONFIG_GEN_SCHEMA_DIR.joinpath("eos_cli_config_gen.schema.yml")
```

Comment:
Why not use the `/` operator on `Path` ?
```suggestion
EOS_CLI_CONFIG_GEN_SCHEMA_PATH = EOS_CLI_CONFIG_GEN_SCHEMA_DIR / "eos_cli_config_gen.schema.yml"
```

Reply:
IMO the method is more clear and readable. You don't have to know that the constant is a `Path` object.